webapp/javascript/components/TimelineChart/TimelineChartSelection.ts

```diff
@@ -389,6 +396,7 @@ const handleHeight = 22;
     plot.clearSelection = clearSelection;
     plot.setSelection = setSelection;
     plot.getSelection = getSelection;
+    plot.setCursor = cursor();
```

Comment:
can you please make name of this function more informative cursor();

plot.setCursor = setCursor(); ?